### ISSUE TITLE
gitshow blurb & scheduling

### ISSUE THREAD

**Issue by michaelpalumbo** [michaelpalumbo/phd#2]
Hi @dvntsea,

Does Wednesday at 9am still work for you to schedule gitshow sessions in the lab? 

Also I edited the blurb that you provided feedback on, [here](https://github.com/michaelpalumbo/phd/blob/master/gitshow/gitshow_blurb.md). 

We just need to settle on a deadline for interested participants. I also think, given the nature of open-source and the cycles of the piece leave room for people to join later on as well (though besides noting this here and writing about it I don’t know if I want to emphasize that too much publicly at the moment).

**Comment by michaelpalumbo** [comment]
@dvntsea I want to submit to MUME 2019, xcoax19(tighter deadline). Should I be clear in the blurb that  my intention is to submit the experiment as a performance and research piece to CFPs for further runs?

**Comment by dvntsea** [comment]
I don’t think we need to mention the ultimate publication/presentation outlet at this stage.

It would be very tight for xcoax so we should start initiating pretty soon.

Should I send the blurb to dispersion announce list and digital media list?

**Comment by michaelpalumbo** [comment]
> I don’t think we need to mention the ultimate publication/presentation outlet at this stage.

sounds good

> It would be very tight for xcoax so we should start initiating pretty soon.

absolutely. is it feasible to begin sessions in 1-2 weeks?

> Should I send the blurb to dispersion announce list and digital media list?

Yes that would be great. Before we do, should we set a deadline or just put it out there to gauge interest?

**Comment by michaelpalumbo** [comment]
Hi @dvntsea, just following up on my last message. 

> >Should I send the blurb to dispersion announce list and digital media list?

> Yes that would be great. Before we do, should we set a deadline or just put it out there to gauge interest?

Thoughts?

Let's schedule the morning improv sessions beginning next week.

**Comment by dvntsea** [comment]
Hi Michael,

1. Please send me the completed blurb via email and i can send it out.

2. There is some confusion. I had said you could have a morning slot — however I did not say I would partake in it and I was thinking Wednesday or Thursday afternoon for gitshow.

Thanks,
Doug

**Comment by michaelpalumbo** [comment]
> 1. Please send me the completed blurb via email and i can send it out.

done!

> 2. There is some confusion. I had said you could have a morning slot — however I did not say I would partake in it and I was thinking Wednesday or Thursday afternoon for gitshow.

ok. so just to clarify for myself, the morning slot is for my own personal gitShow-related work (admin, maintaining the repos, preparing for repo branch transfers from one participant to another, etc)? whereas the wednesday or thursday afternoons would be for the weekly gitshow improv sessions?

Would you still be participating in the weekly improv sessions?

**Comment by michaelpalumbo** [comment]
Do you think its reasonable to begin the 1st cycle of gitshow now, and try to schedule the 1st improv session for next thursday (jan 31)?

**Comment by dvntsea** [comment]
If participants are on board then I don’t see why not.  However my participation is going to have to be tentative for this week.

**Comment by michaelpalumbo** [comment]
OK that's great!
